moveit2.repos

```diff
@@ -15,3 +15,7 @@ repositories:
     type: git
     url: https://github.com/ros-planning/warehouse_ros_mongo
     version: ros2
+  parameter_tools:
```

Comment:
@JafarAbdi This package name is also a bit ambiguous per http://wiki.ros.org/ROS/Patterns/Conventions

Reply:
Do you have a suggestion for a better name?  

The intent of the repo `parameter_tools` is infrastructure for the use of ros2 parameters (similar to rosparam_shortcuts was for ros1).  The reason we wanted to create a new repo is that rosparam_shortcuts was ported to ros2 but I didn't think the approach worked well to solve the new api and pain points of the ros2 parameter API.  Secondly, rosparam_shortcuts is one ros package in a repo and I wanted to have a repo with multiple ros packages as what Jafar created (parameter_builder) is a python package and what I'm working on (parameter_sets) is a C++ package that solve separate problems around the ros2 parameters.

I didn't want to conflict with the rosparam_shortcuts name and wanted a repo that contained tools for making ros2 paramters better.  A big reason why I don't want to put it in moveit2 itself is to avoid creating yet another circular dependency similar to the issues with the moveit_resources package.  moveit2 config packages (like moveit_resources) will need to depend on this package so it would be much nicer (for release testing) if this code is in a separate repo from moveit2.